.github/label-actions.yml

```
@@ -438,4 +454,23 @@
     If you are a paying Mend.io customer, please tell your support or customer contact that this issue is important to you.
 
 
+    Thanks, the Renovate team
+
+'auto:reduce-complexity':
+  comment: >
+    Hi there,
+
+
+    This discussion is too complex, and we recommend you simplify it to increase chances of getting help and finding a solution.
+
+
+    For example, if you've included your _whole_ config, and it's quite complex, while your problem relates to just one specific part, consider removing the parts that are not relevant to the problem. The best way to do this is to usually create a minimal reproduction repository.
+
+
+    You may have also tried many ways to achieve something, and described them all here, but if none of them work then it's best to focus on the one that you think is the most promising or the ideal solution and not complicate the description or logs with the other attempts.
```

```suggestion
    You may have tried many ways to do something, and described all the methods you tried. If none of the methods worked, please focus on the most promising method, or the ideal solution. Avoid complicating the description (or logs) with the failed attempts.
```

Chopping up this sentence into smaller parts.